libvast/vast/fbs/utils.hpp

```diff
@@ -27,7 +30,10 @@
 
 #include <flatbuffers/flatbuffers.h>
 
+#include <algorithm>
 #include <cstdint>
+#include <ctime>
+#include <dirent.h>
```

Comment:
What's up with the new includes? I don't think `fbs/utils.hpp` should include the accountant, or `type.hpp`, or `dirent.h`, whatever that is.